src/Notifier.php

```diff
@@ -96,6 +86,53 @@ public function notify($event, AccountInterface $account = NULL, $usDelay = 0) {
   }
 
   /**
+   * Meteor url validation.
+   *
+   * From https://gist.github.com/dperini/729294
+   *
```

Comment:
Need to add  licensing info:
// Author: Diego Perini
// Updated: 2010/12/05
// License: MIT
//
// Copyright (c) 2010-2013 Diego Perini (http://www.iport.it)